Reachability.m

```diff
@@ -98,8 +98,9 @@ +(instancetype)reachabilityWithHostname:(NSString*)hostname
     SCNetworkReachabilityRef ref = SCNetworkReachabilityCreateWithName(NULL, [hostname UTF8String]);
     if (ref) 
     {
-        id reachability = [[self alloc] initWithReachabilityRef:ref];
-
+        id reachability = [[self alloc] initWithReachabilityRef:CFBridgingRetain((__bridge id)ref)];
+	CFRelease(ref);
```

Comment:
Ah, I see what you did. I missed it and I assumed you did what others did before. 

You can disregard my above comments.

However, I'd like to leave 2 cents if you don't mind :)

Not sure about everyone else but for me your fix is a little hard to read. I totally glanced over your retain statement. Since the ultimate (final) release is in the dealloc, I think many people would expect the retain to happen in the init. Instead of all the helpers that call the init. And while there may not be additional helpers being added any time soon, if someone does, they'll have to be sure to make the retain in their new helper. Since the current helpers are class methods, it does feel a little odd its doing memory management on an instance item.

Again, just my two cents & it may be a little nit-picky, but feels a little anti-pattern.